metabase/metabase
# ci(sdk): embedding e2e workflow fixes for 52 branch

Ensures both the SDK E2E cross-version test workflow and the SDK E2E workflow works in the 52 branch, by ensuring module resolution detects both installed npm packages in `node_module` for e2e cross-version tests, as well as for locally-built `resources/embedding-sdk` packages for regular sdk e2e workflow.

### How to verify

Check out commit [`c855640`'s checks](https://github.com/metabase/metabase/actions/runs/12350994209/job/34465409153?pr=51268) which modifies the workflow file to run against this branch.

You can see in the checks that both E2E and cross-version workflow passes using this module resolution method. Click on "Run details > Workflow file" in the above run to see how I made it run against this PR, you can do a search on "FIXME:" in the logs.
